app/views/controls/views/gradients.xml

```diff
@@ -22,12 +22,15 @@
             <Button onClick="pickRandomColor" class="colorPickerButton" title="🔄 Pick Random Color" />
 
             <!-- Radial / linear -->
-            <TabbedBar class="gradientTypesBar" onClick="handleGradientType">
+            <TabbedBar class="gradientTypesBar" onClick="handleGradientType" platform="ios">
                 <Labels>
                     <Label>Radial</Label>
                     <Label>Linear</Label>
                 </Labels>
             </TabbedBar>
+
+            <Switch id="gradientTypeSwitch" onClick="handleGradientType" platform="android" />
+            <Label id="gradientTypeLabel" platform="android">Radial</Label>
 		</View>
```

Comment:
Turned on editor whitespace and saw the problems, I was using tabs. I've updated the file to use spaces everywhere instead of tabs (like in the rest of the project)